archivebox/archive_methods.py

```diff
@@ -226,7 +227,7 @@ def fetch_wget(link_dir, link, timeout=TIMEOUT):
         '--span-hosts',
         '--no-parent',
         '-e', 'robots=off',
-        '--restrict-file-names=windows',
+        '--restrict-file-names=nocontrol',
```

Comment:
this doesn't work well on macOS unfortunately. can you make it a config option instead?

Reply:
I will create a config option "RESTRICT_FILE_NAMES".